backend/src/main/nl/tudelft/ti2806/riverrush/game/GameTrack.java

```diff
@@ -22,32 +26,30 @@
 
     private static final long UPDATE_DELAY = 1000;
     public static final Integer TRACK_LENGTH = 100;
-    private static final Integer DISTANCE_INTERVAL = 5;
```

Comment:
Does removing this mean we can no longer easily set the interval of events easily?